### PR TITLE
add new words

### PR DESCRIPTION
**Pull request by dpcairns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # some-new-name
 
+# I am on a new branch. I want to make a pull request! So that i can use the normal process that every team on the planet uses when working in github!
```
Comment (dpcairns) [inline README.md:3]: I don't know if this change is a good idea

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # some-new-name
```
Comment (sarah-svg) [inline README.md:2]: hello

Comment (CadillacJack42) [inline README.md:2]: comment

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
 
+# I am on a new branch. I want to make a pull request! So that i can use the normal process that every team on the planet uses when working in github!
 # Big header
 ## Medium header
```
Comment (clayton-knapp) [inline README.md:5]: commenting

File: README.md (hunk 2 above)
Comment (dpcairns) [reply README.md:2]: hi!

**Review by clayton-knapp — APPROVED** [review]

**Pull request merged**